.github/workflows/ghcr_core_master.yml

```diff
@@ -24,7 +24,8 @@ jobs:
           password: "${{ secrets.FLYTE_BOT_PAT }}"
           image_name: ${{ github.repository_owner }}/flytecookbook
           image_tag: latest,core-latest,core-${{ github.sha }},core-${{ github.event.ref }}
-          push_git_tag: true
+          push_git_tag: ${{ github.event.pull_request.head.repo.full_name == github.repository }}
```

Comment:
I don't think `github.event.pull_request` will be there when we merge to master.. it'll only exist in PRs...